packages/smooth_app/lib/views/bottom_sheet_views/user_contribution_view.dart

```diff
@@ -7,13 +7,7 @@ import 'package:smooth_app/generic_lib/buttons/smooth_action_button.dart';
 import 'package:smooth_app/generic_lib/dialogs/smooth_alert_dialog.dart';
 import 'package:smooth_app/generic_lib/widgets/smooth_list_tile.dart';
 import 'package:smooth_app/helpers/launch_url_helper.dart';
-
-class ListHelper {
-  ListHelper({required this.onTap, required this.title, this.icon});
-  String title;
-  void Function() onTap;
-  Icon? icon;
-}
+import '../../helpers/list_helper.dart';
```

Comment:
Somehow also my IDE started using relative paths as default, but yes lets keep the direct way

Reply:
Sure.